src/style.css

```diff
@@ -0,0 +1,101 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  margin-top: 20px;
+}
+
+html {
+  font-family: Avenir, serif;
+  color: #3d4e61;
+}
+
+.searchForm {
+  width: 100%;
+  display: flex;
+  position: relative;
+  background: white;
+  border: 1px solid #e1e7ed;
+  border-radius: 5px;
+}
+
+.searchFormLong {
+  margin-bottom: 20px;
+}
+
+.searchForm .searchFormInput {
```

Comment:
don't increase specificity for selectors
can you leave just `searchFormInput`?

Reply:
fixed selector.
fixed demo link.